modules/local/sentieon/locuscollector.nf

```diff
@@ -0,0 +1,46 @@
+process SENTIEON_LOCUSCOLLECTOR {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+
+    output:
+    tuple val(meta), path('*score.txt.gz')          , emit: score        , optional: true
+    tuple val(meta), path('*score.txt.gz.tbi')      , emit: score_idx    , optional: true
+    path  "versions.yml"                            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
+    def input  = bam ? '-i ' + bam.sort().join(' -i ') : ''
+    """
+    sentieon \\
+        driver \\
+        --algo LocusCollector \\
+        --fun score_info ${idSample}_score.gz \\
```

Comment:
- idSample? (meta.id) 
- output name doesn't match the pattern given in output section